src/Bridge/Doctrine/Orm/CollectionDataProvider.php

```diff
@@ -36,7 +36,7 @@ class CollectionDataProvider implements ContextAwareCollectionDataProviderInterf
     /**
      * @param QueryCollectionExtensionInterface[]|ContextAwareQueryCollectionExtensionInterface[] $collectionExtensions
      */
-    public function __construct(ManagerRegistry $managerRegistry, array $collectionExtensions = [])
+    public function __construct(ManagerRegistry $managerRegistry, \iterable $collectionExtensions = [])
```

Comment:
Please, don't, we also have to support PHP 7.0!

Reply:
bah, forgot about that!

Reply:
ah, forgot about that!